### PR TITLE
Bump version to 0.1.33

### PR DESCRIPTION
**Pull request by zanieb**

### CODE REVIEW

**Review by charliermarsh — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,40 @@
...
+### Performance
+
+- Incorporate heuristics to improve package prioritization ([#3087](https://github.com/astral-sh/uv/pull/3087))
+
+### Bug fixes
+
+- Fix HTTP authentication when the password includes percent encoded characters (e.g. with Google Artifact Registry) ([#2822](https://github.com/astral-sh/uv/issues/2822))
+- Use usernames from URLs when looking for credentials in netrc files and the keyring [#2563](https://github.com/astral-sh/uv/issues/2563))
+- Skip `HEAD` requests for indexes that return 403 (e.g. PyPICloud) ([#3070](https://github.com/astral-sh/uv/pull/3070))
+- Fix reference to `pyproject.toml` path in workspace discovery ([#3083](https://github.com/astral-sh/uv/pull/3083))
```
Comment (konstin) [inline CHANGELOG.md:33]: Is that public yet?

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,40 @@
...
+was assumed. This will affect Google Artifact Registry users using `--keyring-provider subprocess` and an index URL
+without a username. The suggested fix is to add the required username to index URLs, e.g., `https://oauth2accesstoken@<url>`.
+
+See [#2976](https://github.com/astral-sh/uv/pull/2976#discussion_r1566521453) for details.
+
+### Enhancements
+
+- Allow passing a virtual environment path to `uv pip --python` ([#3064](https://github.com/astral-sh/uv/pull/3064))
+- Add compatibility argument for `pip list --outdated` ([#3055](https://github.com/astral-sh/uv/pull/3055))
+- Add support for `.tar.bz2` source distributions ([#3069](https://github.com/astral-sh/uv/pull/3069))
```
Comment (konstin) [inline CHANGELOG.md:17]: Note that this is a fallback, we don't support-support (https://packaging.python.org/en/latest/specifications/source-distribution-format/#source-distribution-file-name) these but just don't error anymore when they exist in a legacy registry. Uploading to pypi is banned (https://github.com/pypi/warehouse/blob/ae35f55aa9ae6250e7a698e34dbe64fbbf54632b/tests/unit/forklift/test_legacy.py#L1556)

**Review by konstin — APPROVED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (zanieb) [reply CHANGELOG.md:33]: Nope!

File: CHANGELOG.md (hunk 2 above)
Comment (zanieb) [reply CHANGELOG.md]: Thanks!

**Pull request merged**